recipes-tag/bluesky/meta.yaml

```diff
@@ -1,43 +1,59 @@
+{% set name = "bluesky" %}
 {% set version = "1.1.0" %}
+{% set sha256 = "bf5467874716eca7741d8a95326317fa2ae3778c4211cb7b52588a830c62169e" %}
 
 package:
-    name: bluesky
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/nsls-ii/bluesky
-    git_rev: v{{ version }}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NSLS-II/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record=record.txt
-    skip: True  # [py2k]
+  number: 0 
+  skip: True  # [py2k or osx]
```

Comment:
Why skip OSX?

Reply:
Travis was so behind that it was not worth building on OSX. As things speed up it may get better and I'm more than happy to build on all platforms https://www.traviscistatus.com/